### PR TITLE
Release fixes

### PR DESCRIPTION
**Pull request by jglick**
Trying to make this releasable to OSSRH.

Some temporary POM code pending merge & release of https://github.com/kohsuke/pom/pull/1.

@reviewbybees esp. @kohsuke and @tfennelly

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -171,6 +170,30 @@
         <version>1.2</version>
       </plugin>
     </plugins>
+    <pluginManagement>
+      <plugins>
```
Comment (jtnord) [inline pom.xml]: there is a site distribution URL so may also need the site plugin and associated bits.

**Comment by tfennelly** [comment]
:+1: 

Curious ... do we actually need `<pre>... </pre>` wrapping `{@code ... }` ? Crazy that javadoc has an issue with `<pre>`.

**Comment by tfennelly** [comment]
oh and ... thanks Jesse !!

File: pom.xml (hunk 1 above)
Comment (jglick) [inline pom.xml]: Maybe. I merely corrected the Maven warnings turned up by `clean install`. There might be other warnings displayed during `release:{prepare,perform}` that I did not see yet; AFAIK there is no good way to preview that (one of the reasons `maven-release-plugin` is a dumb design).

**Comment by jglick** [comment]
> do we actually need `<pre>... </pre>` wrapping `{@code ... }` ?

Not positive. Inline Javadoc preview in the NetBeans IDE seemed to require it. I did not bother to check against `javadoc:javadoc` generated HTML. I picked up the idiom [here](http://minds.coremedia.com/2012/05/16/javadoc-hassle-free-embedded-code-using-code/). Anyway it seems harmless.

**Comment by stephenc** [comment]
I have reservations about the level of maven abuse going on here, but given that we want to unblock the release...

:+1:

**Comment by jglick** [comment]
> maven abuse

Alternate suggestions?

**Comment by stephenc** [comment]
>>maven abuse
>
>Alternate suggestions?

Oh it involves reworking custom packaging etc. Nothing quick

**Pull request merged**